examples/experimental/i3s-picking/mesh-layer/matrix.js

```diff
@@ -0,0 +1,166 @@
+import {COORDINATE_SYSTEM, createIterable} from '@deck.gl/core';
```

Comment:
Since we are copying in so much code, it is getting hard to understand what is just being copied in and what is modified.

I don't know what to review.

Perhaps we can clearly mark any changes we make with comments before and after.